frontend/src/components/Header/Header.js

```diff
@@ -41,6 +42,10 @@ const Header = () => {
     </div>
   );
 
+  const fullName = useSelector(
+    (state) => state.login.user.firstName + ' ' + state.login.user.lastName
```

Comment:
Destructure `state` in the argument declarations such that your string concatenation is only `firstName + ' ' + lastName` 

https://medium.com/@pyrolistical/destructuring-nested-objects-9dabdd01a3b8

Reply:
done

Reply:
done